.changeset/config.json

```diff
@@ -12,5 +12,5 @@
   "access": "public",
   "baseBranch": "main",
   "updateInternalDependencies": "patch",
-  "ignore": []
+  "ignore": ["@examples/next-advanced", "@examples/next-simple"]
```

Comment:
Can't we have a wildcard ?

Reply:
@chambo-e I replaced by: `"ignore": ["@examples/*"]`